docs/prerequisites.md

```diff
@@ -0,0 +1,31 @@
+# Prerequisites
+
+## Clone
+
+This repository is **independent of CoreFX and CoreCLR repositories!**  So this is the only repository you need to clone.
+
+```cmd
+git clone https://github.com/dotnet/performance.git
+```
+
```

Comment:
Maybe we need a section here for the actual prerequisites?

## External tools

### [.NET Core command-line interface (CLI) tools](https://docs.microsoft.com/en-us/dotnet/core/tools/?tabs=netcore2x)

Maybe explain here why is needed or where it is used?
- [.NET Downloads](https://dotnet.microsoft.com/download)

### [python](https://www.python.org/)

Maybe explain here why is needed or where it is used?
- [python Downloads](https://www.python.org/downloads/)
  - NOTE: The python scripts in this repository support python version 3.5 or greater.

Reply:
ping.